docs/content/docs/ops/state/checkpoints.md

```diff
@@ -175,6 +175,35 @@ above).
 $ bin/flink run -s :checkpointMetaDataPath [:runArgs]
 ```
 
+### Checkpoint/Savepoint guarantees
```

Comment:
Should we combine it with the section "Difference to Savepoints"?

Reply:
I wonder whether it should be in its own page titled "Guarantees" or something like that.  But I am not sure it fits into the Checkpoints page as it currently stands.  I guess we could put it into "Difference to Savepoints" but then the table also has pretty important information and maybe it would stand out more as its own page.

Reply:
Yes, I also thought about the extra page for this information but I also wasn't sure to worth it or not.